salt/cloud/clouds/vmware.py

```diff
@@ -3056,7 +3056,7 @@ def create_snapshot(name, kwargs=None, call=None):
         log.warning('You can only set either memdump or quiesce to True. Setting quiesce=False')
         quiesce = False
 
-    desc = kwargs.get('description') if 'description' in kwargs else ''
+    desc = kwargs.get('description') if kwargs and 'description' in kwargs else ''
```

Comment:
@rallytime Like I said above, this isn't needed. 

``desc = kwargs.get('description', '')`` should be sufficient.